src/main/java/org/jilt/internal/AbstractBuilderGenerator.java

```diff
@@ -113,6 +127,24 @@ public final void generateBuilderClass() throws Exception {
         javaFile.writeTo(filer);
     }
 
+    private static boolean getLombokBuilderDefaultAnnotation(List<? extends AnnotationMirror> annotationMirrors) {
+        if (annotationMirrors == null || annotationMirrors.isEmpty()) {
+            return false;
+        }
+        return annotationMirrors
+                .stream()
+                .anyMatch(a -> "lombok.Builder.Default".equals(a.getAnnotationType().asElement().toString()));
+    }
+
+    private CodeBlock getCodeBlockInitializer(VariableElement attribute) {
+        Tree tree = trees.getPath(attribute).getLeaf();
+        String initializer = "";
+        if (tree instanceof VariableTree && ((VariableTree) tree).getInitializer() != null) {
+            initializer = ((VariableTree) tree).getInitializer().toString();
+        }
+        return initializer.isEmpty()? CodeBlock.of("") : CodeBlock.of("$L", initializer);
```

Comment:
Yes indeed, I could make it fail if there's no initialization. 

If you know of an easier way to do this, I'd love to hear about it, but I've been doing a lot of searching on the internet and came across this. 
getLeaf retrieves the syntactic representation of the variable. VariableTree represents the variable's syntactic node.

Unfortunately, I couldn't find a simpler way of doing this, and the problem with this code is that to use Tree tree = trees.getPath(attribute).getLeaf();, you need to be in a java version higher than java 8 (I wasn't able to test in java 9).

Reply:
OK. For now, let's just leave it as-is. We'll figure out how to deal with this later (most likely, we'll reproduce whatever this does, just only for fields, since we don't care about other element kinds).

Can you remove Java 8 from [here](https://github.com/skinny85/jilt/blob/5566380bffb7a4234d03167a7c35840df64eb265/.github/workflows/build.yaml#L10-L10) temporarily, so that we can see whether the build passes for Java 9+?